builder/src/build/patches/investigated/update-webpack-chunks-file/get-chunk-installation-identifiers.test.ts

```diff
@@ -0,0 +1,30 @@
+import { readFile } from "node:fs/promises";
+
+import { expect, test, describe } from "vitest";
+
+import { getChunkInstallationIdentifiers } from "./get-chunk-installation-identifiers";
+import { getWebpackChunksFileTsSource } from "./get-webpack-chunks-file-ts-source";
+
+describe("getChunkInstallationIdentifiers", () => {
+  test("works as expected on unminified code", async () => {
```

Comment:
nit: IMO "works as expected on " can be dropped - "get chunk identifiers on unminified code" would be more informative

Reply:
updated 🙂👍 

Please have another look 🙏